test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -505,22 +503,20 @@ export default class PerfCompare extends Component {
                     }
                 } catch (higherBetterNotFoundError) {
 
-                    curHigherBetter = undefined;
+                    curHigherBetter = true;
```

Comment:
We talked about this in the previous PR. Instead of try catch, please change it into one line.